content/en/cloud_cost_management/cost_allocation/bigquery.md

```diff
@@ -0,0 +1,112 @@
+
+**Idle slot sharing considerations**: If your organization has enabled idle slot sharing between reservations, the idle cost calculation may appear different than expected. When queries from one project use idle slots from another project's reservation, those slot costs are attributed as "free" rather than to the consuming project. This means:
+
+- A project's reservation may show higher idle costs if other projects are using its unused capacity
+- The original project pays full reservation costs regardless of cross-project usage
+- No automatic cost-transfer: Sharing projects don't pay the reservation owner for consumed idle slots
+
+### Query-level tag extraction
+
+CCM extracts the following tags from BigQuery query logs via the Data Observability platform:
+
+| Tag | Description |
+|---|---|
+| `reservation_id` | The reservation pool that provided compute resources |
+| `user_email` | The user or service account that executed the query |
+| `dts_config_id` | Identifier for scheduled queries and data transfers |
+
+Additionally, CCM adds the following tags for cost analysis:
+
+| Tag | Description |
+|---|---|
+| `allocated_spend_type` | Categorizes costs as either `usage` (active query execution) or `cluster_idle` (unused reservation capacity) |
+| `allocated_resource` | Indicates resource measurement type - `slots` for reservation-based queries or `bytes_processed` for on-demand queries |
+| `orchestrator` | Set to `BigQuery` for all BigQuery query-related records |
+
+### Compute allocation
+
+For BigQuery compute allocation, CCM handles two pricing models:
+
+**On-demand pricing**:
+- Costs are directly attributed to individual queries based on bytes processed
+- Includes query-level tags for detailed cost attribution
+
+**Reservation-based pricing**:
+- Costs of reserved slots are allocated proportionally to queries using those slots
+- Allocation based on slot consumption (`total_slot_ms`) per query
+- Includes idle cost calculation for unused reservation capacity
+
+### Compute
+
+Costs are allocated into the following spend types:
+
+| Spend type | Description |
+|---|---|
+| Usage | Cost of query execution based on bytes processed (on-demand) or slot consumption (reservation) |
+| Cluster_idle | Cost of reserved slots allocated within a project but not utilized by queries|
+
+### Storage
+
+Storage costs are categorized as:
+
+| Spend type | Description |
+|---|---|
+| Active | Includes any table or table partition that has been modified in the last 90 days |
+| Long-term | Includes any table or table partition that has not been modified for 90 consecutive days. The price of storage for that table automatically drops by approximately 50%. There is no difference in performance, durability, or availability between active and long-term storage |
+
+## Further reading
+ 
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]: https://app.datadoghq.com/dashboard/ecm-es8-agw/bigquery-allocation
```

Comment:
Lets make sure we do not merge this PR until this dashboard is pushed to production and the link is corrected to the version that works in everyone's account. Otherwise this is just a link to the internal test version of the dashboard